bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/template/TemplateRegistry.java

```diff
@@ -26,7 +26,6 @@
  * @author Yordan Mihaylov - Initial Contribution
  * @author Ana Dimova - Initial Contribution
  * @author Vasil Ilchev - Initial Contribution
- * @param <T>
```

Comment:
I agree that the removal of `@param <T>` is correct, but there should be a `@param <E>` inserted instead.